diff --git a/dist/index.js b/dist/index.js
index 2d4658a764b4d663b57f1ed0f5d97cd114674a17..d79ba95e8820fe71bc1d8ce709052d86319c8207 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -1,5 +1,5 @@
 // src/debug.tsx
-import { jsx, jsxs } from "react/jsx-runtime";
+import { jsx, jsxs } from "react/jsx-runtime.js";
 function Debug(props) {
   const { debug, ...rest } = props;
   return /* @__PURE__ */ jsxs("details", { ...rest, children: [
@@ -10,7 +10,7 @@ function Debug(props) {
 
 // src/compose.tsx
 import React from "react";
-import { Fragment, jsx as jsx2 } from "react/jsx-runtime";
+import { Fragment, jsx as jsx2 } from "react/jsx-runtime.js";
 function Compose({ elements }) {
   return elements.reduceRight(
     (acc, el) => React.cloneElement(el, {}, acc),