diff --git a/dist/vite/plugin.js b/dist/vite/plugin.js
index 4d7d30ef9079e2acf0b9317a9b8245467f232149..577b1df11f5a1496c618d85fb25b7dc8579bae12 100644
--- a/dist/vite/plugin.js
+++ b/dist/vite/plugin.js
@@ -23,7 +23,7 @@ var pick = require('lodash/pick');
 var pc = require('picocolors');
 var config = require('../config.js');
 var invariant = require('../invariant.js');
-var adapter = require('./node/adapter.js');
+// var adapter = require('./node/adapter.js');
 var styles = require('./styles.js');
 var vmod = require('./vmod.js');
 var removeExports = require('./remove-exports.js');