diff --git a/dist/vite/node/adapter.js b/dist/vite/node/adapter.js
index c6f7097c1257ff59652836db54426030120881de..d49d41b9e185275692be312dc26390a2cf4ec9f6 100644
--- a/dist/vite/node/adapter.js
+++ b/dist/vite/node/adapter.js
@@ -25,7 +25,8 @@ function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'defau
 var multipart__default = /*#__PURE__*/_interopDefaultLegacy(multipart);
 
 // @ts-nocheck
-node.installGlobals();
+// imcompatibility with hattip on nodejs?
+// node.installGlobals();
 function nodeToWeb(nodeStream) {
   let destroyed = false;
   let listeners = {};
diff --git a/dist/vite/plugin.js b/dist/vite/plugin.js
index 2a9bc7066ecd3819b9dd490d6a8cb543682cdbfa..61d5b8013ba54917005499dec70b098c342355e4 100644
--- a/dist/vite/plugin.js
+++ b/dist/vite/plugin.js
@@ -782,7 +782,7 @@ const remixVitePlugin = (options = {}) => {
 };
 function addRefreshWrapper(pluginConfig, code, id) {
   let isRoute = getRoute(pluginConfig, id);
-  let acceptExports = isRoute ? ["meta", "links", "shouldRevalidate"] : [];
+  let acceptExports = isRoute ? ["handle", "meta", "links", "shouldRevalidate"] : [];
   return REACT_REFRESH_HEADER.replace("__SOURCE__", JSON.stringify(id)) + code + REACT_REFRESH_FOOTER.replace("__SOURCE__", JSON.stringify(id)).replace("__ACCEPT_EXPORTS__", JSON.stringify(acceptExports));
 }
 const REACT_REFRESH_HEADER = `