diff --git a/dist/server.js b/dist/server.js
index fd10bc563175ba69bda9b21ccc7732b20f794900..ebb6e94785bee2bca35031977d083ad516a3ae94 100644
--- a/dist/server.js
+++ b/dist/server.js
@@ -29,6 +29,8 @@ const createRequestHandler = (build, mode$1) => {
   let serverMode = mode.isServerMode(mode$1) ? mode$1 : mode.ServerMode.Production;
   let staticHandler = router.createStaticHandler(dataRoutes);
   return async function requestHandler(request, loadContext = {}) {
+    // see misc/initialize-server.ts
+    await globalThis.__onRequestHandler?.();
     let url = new URL(request.url);
 
     // special __REMIX_ASSETS_MANIFEST endpoint for checking if app server serving up-to-date routes and assets