diff --git a/server.js b/server.js
index a5ddec23cb0eb4cfbe5d60b3167456ae2f7b1fbb..e1021ed9b9be645c65eaad4e2be6217a043bc608 100644
--- a/server.js
+++ b/server.js
@@ -26,6 +26,8 @@ const createRequestHandler = (build, mode$1) => {
   let routes$1 = routes.createRoutes(build.routes);
   let serverMode = mode.isServerMode(mode$1) ? mode$1 : mode.ServerMode.Production;
   return async function requestHandler(request, loadContext) {
+    // see misc/initialize-server.ts
+    await globalThis.__onRequestHandler?.();
     let url = new URL(request.url);
     let matches = routeMatching.matchServerRoutes(routes$1, url.pathname);
     let response;