diff --git a/dist/vite/plugin.js b/dist/vite/plugin.js
index 4d7d30ef9079e2acf0b9317a9b8245467f232149..b437ac3a61f3bf0a8d9dae4b6819a3d91459c7cd 100644
--- a/dist/vite/plugin.js
+++ b/dist/vite/plugin.js
@@ -23,7 +23,7 @@ var pick = require('lodash/pick');
 var pc = require('picocolors');
 var config = require('../config.js');
 var invariant = require('../invariant.js');
-var adapter = require('./node/adapter.js');
+// var adapter = require('./node/adapter.js');
 var styles = require('./styles.js');
 var vmod = require('./vmod.js');
 var removeExports = require('./remove-exports.js');
@@ -657,7 +657,7 @@ const remixVitePlugin = (options = {}) => {
           allowAwaitOutsideFunction: true,
           plugins: ["jsx", "typescript"]
         },
-        plugins: [require("react-refresh/babel")],
+        plugins: [[require("react-refresh/babel"), { skipEnvCheck: true }]],
         sourceMaps: true
       });
       if (result === null) return;
@@ -704,7 +704,7 @@ const remixVitePlugin = (options = {}) => {
 };
 function addRefreshWrapper(pluginConfig, code, id) {
   let isRoute = getRoute(pluginConfig, id);
-  let acceptExports = isRoute ? ["meta", "links", "shouldRevalidate"] : [];
+  let acceptExports = isRoute ? ["meta", "links", "shouldRevalidate", "handle"] : [];
   return REACT_REFRESH_HEADER.replace("__SOURCE__", JSON.stringify(id)) + code + REACT_REFRESH_FOOTER.replace("__SOURCE__", JSON.stringify(id)).replace("__ACCEPT_EXPORTS__", JSON.stringify(acceptExports));
 }
 const REACT_REFRESH_HEADER = `