diff --git a/dist/index.js b/dist/index.js
index 07e6f6cb0776411bb6751ebe6dfd6fd9b4216c6a..2e1ea824afbf5c8b973bfe22bae26e0a90baeb67 100644
--- a/dist/index.js
+++ b/dist/index.js
@@ -27,7 +27,7 @@ function _interopDefaultLegacy (e) { return e && typeof e === 'object' && 'defau
 
 var sourceMapSupport__default = /*#__PURE__*/_interopDefaultLegacy(sourceMapSupport);
 
-sourceMapSupport__default["default"].install();
+// sourceMapSupport__default["default"].install();
 
 Object.defineProperty(exports, 'AbortController', {
   enumerable: true,