diff --git a/argon2.js b/argon2.js
index 65ac4b1f61537cb758d933b7795276faaaee3a55..6a2d82495654c866e163e8b5289d78d820270e4c 100644
--- a/argon2.js
+++ b/argon2.js
@@ -3,9 +3,20 @@ const assert = require("assert");
 const { randomBytes, timingSafeEqual } = require("crypto");
 const path = require("path");
 const { promisify } = require("util");
-const binary = require("@mapbox/node-pre-gyp");
 
-const bindingPath = binary.find(path.resolve(__dirname, "./package.json"));
+// (patch)
+// load prebuilt "argon2.node" in working directory
+// to skip "@mapbox/node-pre-gyp" dependency since it's not bundler-friendly
+
+let bindingPath;
+if (require("fs").existsSync("argon2.node")) {
+  bindingPath = path.resolve("argon2.node");
+
+} else {
+  // "slice" to hide it from bundler
+  const binary = require("@mapbox/node-pre-gyp".slice());
+  bindingPath = binary.find(path.resolve(__dirname, "./package.json"));
+}
 const { hash: _hash } = require(bindingPath);
 
 const { deserialize, serialize } = require("@phc/format");