diff --git a/node_modules/@remix-run/dev/compiler.js b/node_modules/@remix-run/dev/compiler.js
index 0de2694..a162f9a 100644
--- a/node_modules/@remix-run/dev/compiler.js
+++ b/node_modules/@remix-run/dev/compiler.js
@@ -331,7 +331,7 @@ async function createBrowserBuild(config, options) {
     bundle: true,
     logLevel: "silent",
     splitting: true,
-    sourcemap: options.sourcemap,
+    sourcemap: options.sourcemap ? "inline" : false,
     metafile: true,
     incremental: options.incremental,
     mainFields: ["browser", "module", "main"],
@@ -375,6 +375,7 @@ function createServerBuild(config, options, assetsManifestPromiseRef) {
     absWorkingDir: config.rootDirectory,
     stdin,
     entryPoints,
+    external: ["mysql"],
     outfile: config.serverBuildPath,
     write: false,
     conditions: isCloudflareRuntime ? ["worker"] : undefined,
@@ -389,7 +390,7 @@ function createServerBuild(config, options, assetsManifestPromiseRef) {
     bundle: true,
     logLevel: "silent",
     incremental: options.incremental,
-    sourcemap: options.sourcemap,
+    sourcemap: options.sourcemap ? "inline" : false,
     // use linked (true) to fix up .map file
     // The server build needs to know how to generate asset URLs for imports
     // of CSS and other files.
diff --git a/node_modules/@remix-run/dev/compiler/plugins/serverBareModulesPlugin.js b/node_modules/@remix-run/dev/compiler/plugins/serverBareModulesPlugin.js
index dede8a5..1cbcbe4 100644
--- a/node_modules/@remix-run/dev/compiler/plugins/serverBareModulesPlugin.js
+++ b/node_modules/@remix-run/dev/compiler/plugins/serverBareModulesPlugin.js
@@ -82,6 +82,7 @@ function serverBareModulesPlugin(remixConfig, dependencies, onWarning) {
           // Always bundle everything for cloudflare.
           case "cloudflare-pages":
           case "cloudflare-workers":
+          case "deno":
             return undefined;
         }
 
