diff --git a/node_modules/knex/lib/knex-builder/make-knex.js b/node_modules/knex/lib/knex-builder/make-knex.js
index 55f9acd..51be552 100644
--- a/node_modules/knex/lib/knex-builder/make-knex.js
+++ b/node_modules/knex/lib/knex-builder/make-knex.js
@@ -1,7 +1,5 @@
 const { EventEmitter } = require('events');
 
-const { Migrator } = require('../migrations/migrate/Migrator');
-const Seeder = require('../migrations/seed/Seeder');
 const FunctionHelper = require('./FunctionHelper');
 const QueryInterface = require('../query/method-constants');
 const merge = require('lodash/merge');
@@ -47,6 +45,7 @@ const KNEX_PROPERTY_DEFINITIONS = {
 
   migrate: {
     get() {
+      const { Migrator } = require('../migrations/migrate/Migrator');
       return new Migrator(this);
     },
     configurable: true,
@@ -54,6 +53,7 @@ const KNEX_PROPERTY_DEFINITIONS = {
 
   seed: {
     get() {
+      const Seeder = require('../migrations/seed/Seeder');
       return new Seeder(this);
     },
     configurable: true,
